### ISSUE TITLE
Add testnet faucet details

### ISSUE THREAD

**Issue by hanniabu** [eth-educators/ethstaker-knowledgebase#19]
https://ethereum.stackexchange.com/questions/149267/how-to-get-enough-sepolia-eth-to-test-staking

I think it'd be good to add this info (holesky also supported) to here: https://docs.ethstaker.cc/ethstaker-knowledge-base/tutorials/solo-staking-guides 

or maybe under getting started?

**Comment by EridianAlpha** [comment]
Yes this is something we should add, I'll let you know when it's been included, thanks!

**Comment by coincashew** [comment]
Don't forget about Ephemery!

Faucets
https://faucet.bordel.wtf/
https://ephemery-faucet.pk910.de/ (PoW)
